lndclient.go

```diff
@@ -347,7 +347,11 @@ func (l *lndclientGrpc) getPendingIncomingHtlcs(ctx context.Context, peer *route
 				htlc:    htlc.HtlcIndex,
 			}
 
-			htlcs[key] = struct{}{}
+			htlcs[key] = &inFlightHtlc{
+				addedTs: time.Now(),
```

Comment:
Is Now() the most accurate thing here to do, or would it be better to signal `0` and make it clear that we don't know how long it is in flight?

Reply:
A zero value timestamp does mean that these HTLCs will be first to go when we clear the DB (since we delete oldest `add_time` first). That doesn't feel like a terrible outcome to me, because it's our least valuable data (given that it's incomplete).

Updated the PR to have zero `addedTs` - I'd be tempted to not store them at all and just log that we're not going to bother with incomplete HTLCs, but perhaps that's overkill?

Reply:
Not storing at all sounds good to me. The tool was down and likely missed other data as well. Also fixes the problem of representing incomplete data in the db cleanly.